### PR TITLE
Add support for BETWEEN

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

**Comment by mfussenegger** [comment]
@matriv please review

**Comment by mfussenegger** [comment]
@dobe please take a look as well

File: blackbox/docs/sql/queries.txt [hunk 1]
```diff
@@ -157,6 +157,7 @@ all simple data types:
     like              matches a part of the given value
     ~                 regular expression match
     !~                negated regular expression match
+    between           shortcut for `a >= x AND a <= y`
```
Comment (dobe) [inline blackbox/docs/sql/queries.txt]: pls describe more what a and x and y is. probably by example?

Comment (mfussenegger) [inline blackbox/docs/sql/queries.txt]: I've added an example

**Comment by mfussenegger** [comment]
We can add symmetric later on I guess - if somebody asks for it.
I mainly did this because I saw that it is a very low hanging fruit. Symmetric would require parser changes as well.

**Pull request merged**